tests/testthat/test_euclidean_space.R

```diff
@@ -0,0 +1,8 @@
+# unit test of euclidean_space.R
```

Comment:
`Unit tests for Euclidean space` to be consistent with the test file of Manifold